src/pages/Setting/typingd.d.ts

```diff
@@ -6,7 +6,7 @@ declare namespace Setting {
     key: string;
   }
 
-  interface DashboardConfig extends AdminAPI {
-    grafanaUrl: string;
+  interface GrafanaURL {
```

Comment:
too wired...

Reply:
?

Reply:
Why the typing GrafanaURL includes a grafanaURL property??